ionos_dev/knowledgebase/bookstack/deployment.yaml

```diff
@@ -68,7 +68,7 @@ spec:
             - name: OIDC_CLIENT_ID
               value: "bookstack"
             - name: OIDC_CLIENT_SECRET
-              value: "4XthWaqj7gpREsj9NJQBsGIprIZqJsmy"
+              value: "K8D329TALnXpXmP8NoMHsIobsR77zRDX"
```

Comment:
Are you aware that this secret is publicly visible to everyone on the internet and therefor unsecure?